packages/projection-typeorm/src/operators/storeGovernanceAction.ts

```diff
@@ -0,0 +1,35 @@
+import { ChainSyncEventType } from '@cardano-sdk/core';
+import { GovernanceActionEntity } from '../entity';
+import { ProjectionEvent } from '@cardano-sdk/projection';
+import { WithTypeormContext } from './withTypeormTransaction';
+import { typeormOperator } from './util';
+
+export const willStoreGovernanceAction = (evt: ProjectionEvent<WithTypeormContext>) =>
+  evt.block.body.some((tx) => tx.body.proposalProcedures?.length);
+
+export const storeGovernanceAction = typeormOperator(async (evt) => {
+  if (evt.eventType === ChainSyncEventType.RollBackward || !willStoreGovernanceAction(evt)) return;
+
+  const { block, queryRunner } = evt;
+  const { body, header } = block;
+  const repository = queryRunner.manager.getRepository(GovernanceActionEntity);
+
+  for (const tx of body)
```

Comment:
The generic mapping part of this should be implemented in https://github.com/input-output-hk/cardano-js-sdk/tree/master/packages/projection/src/operators, since it's not specific to TypeORM. Use an existing projection to see the pattern or check the docs:
https://github.com/input-output-hk/cardano-js-sdk/tree/master/packages/projection#mappers

`store` usually doesn't need to be concerned with blocks.

Reply:
ded3e97e6d1 should solve this